include/openspace/rendering/renderable.h

```diff
@@ -54,6 +54,11 @@ class Camera;
 
 class Renderable : public properties::PropertyOwner {
 public:
+    struct Settings {
+        bool automaticallyUpdateRenderBin = true;
```

Comment:
Is this value stored in the renderable somewhere? It looks like the `Settings` object is destroyed after the construction of the `Renderable` and cannot be accessed after that. For the models, I would like to somehow find out if the user specifically specified a `RenderBin` in the asset file so I won't do the automatic `Renderbin` switching there either